configs/proxy_connect_udp.yaml

```diff
@@ -0,0 +1,59 @@
+# This will proxy CONNECT-UDP requests from a downstream connecting on 127.0.0.1:10001 to an
+# upstream listening on 127.0.0.1:10002. Note that the downstream client uses TCP in this example.
```

Comment:
What does "uses TCP" mean here? Is it "Uses HTTP/1.1 or HTTP/2 but not HTTP/3"? Maybe it makes more sense to use HTTP/3 since that's the most common use for CONNECT-UDP

Reply:
Right, that was what I meant with the comment. Changed the config to use HTTP/3 instead.